test/helpers/fusionUtils.js

```diff
@@ -97,8 +101,56 @@ async function buildAuctionDetails({
     return { gasBumpEstimate, gasPriceEstimate, startTime, duration, delay, initialRateBump, details };
 }
 
+function buildSettlementExtensions({
+    feeTaker,
+    estimatedTakingAmount,
+    getterExtraPrefix = '0x',
+    integratorFeeRecipient = constants.ZERO_ADDRESS,
+    protocolFeeRecipient = constants.ZERO_ADDRESS,
+    makerReceiver = undefined,
+    integratorFee = 0,
+    integratorShare = 50,
+    resolverFee = 0,
+    whitelistDiscount = 50,
+    whitelist = '0x00',
+    whitelistPostInteraction = whitelist,
+    protocolSurplusFee = 0,
+    customMakingGetter = '0x',
+    customTakingGetter = '0x',
+    customPostInteraction = '0x',
+}) {
+    const feeTakerExtensionsObj = buildFeeTakerExtensions({
+        feeTaker,
+        getterExtraPrefix,
+        integratorFeeRecipient,
+        protocolFeeRecipient,
+        makerReceiver,
+        integratorFee,
+        integratorShare,
+        resolverFee,
+        whitelistDiscount,
+        whitelist,
+        whitelistPostInteraction,
+        customMakingGetter,
+        customTakingGetter,
+        customPostInteraction: '0x',
+    });
+
+    const postInteraction = ethers.solidityPacked(
```

Comment:
Should we use whitelist here?

Reply:
we are using default behavior for whitelist